simpeg/electromagnetics/natural_source/receivers.py

```diff
@@ -627,3 +1188,25 @@ def evalDeriv(self, src, mesh, f, du_dm_v=None, v=None, adjoint=False):
         if adjoint:
             return imp_deriv
         return getattr(imp_deriv, self.component)
+
+
+@deprecate_class(removal_version="0.20.0", error=False)
+class PointNaturalSource(PointImpedance):
+    """This class is deprecated and will be removed in simpeg v0.20.0.
+    Please use :class:`.PointImpedance`."""
```

Comment:
If we deprecate this function we should remove it at least in simpeg v0.23.
```suggestion
    """This class is deprecated and will be removed in simpeg v0.23.0.
    Please use :class:`.PointImpedance`."""
```